tests/klay_test_blockchain_test.go

```diff
@@ -352,6 +352,18 @@ func (bcdata *BCData) GenABlockWithTransactions(accountMap *AccountMap, transact
 	}
 	prof.Profile("main_mineABlock", time.Now().Sub(start))
 
+	txs := make(types.Transactions, len(b.Transactions()))
+	for i, tt := range b.Transactions() {
+		encodedTx, err := rlp.EncodeToBytes(tt)
+		if err != nil {
+			return err
+		}
+		decodedTx := types.Transaction{}
+		rlp.DecodeBytes(encodedTx, &decodedTx)
+		txs[i] = &decodedTx
+	}
+	b = b.WithBody(txs)
```

Comment:
Why do we need these lines?

Reply:
Without this, we cannot validate the transaction sender again because it is cached by senderCacher.
I wanted to reset the cached sender so that sender validation will be proceeded again.

Reply:
Please note that this is for testing.